### PR TITLE
APIGOV-24164: Add Public Docs for Authorization to Platform APIs

### PR DESCRIPTION
**Pull request by chrisccoy**
## Added new markdown for Authorization

Added new markdown for Authorization and supplemental images of screenshots

## Checklist for contributors

Before submitting this PR, please make sure:

* [ x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ x] You have verified the technical accuracy of your change
* [x ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

### CODE REVIEW

**Review by robertengaxway — APPROVED** [review]
I approve and I would suggest placing it in the "Integrate with Amplify" section

File: content/en/docs/integrate_with_central/platform-auth-examples.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+title: How to authorize API Calls to Platform Services
+---
+Step by step guide for authorizing clients to make REST calls to the Amplify Platform API. This section will outline the easiest way, by using a service account client and secret key. There is also a private key method that will not be covered in the section.
```
Comment (ingo) [inline content/en/docs/integrate_with_central/platform-auth-examples.md:4]: If we mention a different method, we should indicate where it *is* covered or likely delete it for now.

Comment (ingo) [inline content/en/docs/integrate_with_central/platform-auth-examples.md:4]: *A step-by-step*...

File: content/en/docs/integrate_with_central/platform-auth-examples.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+## Prerequisite - API authorization
+
+1. ***Install the Axway CLI***: [https://docs.axway.com/bundle/axwaycli-open-docs/page/docs/quick\_start/index.html](https://docs.axway.com/bundle/axwaycli-open-docs/page/docs/quick_start/index.html)
+
+2. ***Install JQ***:  [https://github.com/stedolan/jq](https://github.com/stedolan/jq)
+
+JQ is needed for parsing the authorization response and extracting the bearer token and org id required in the http  request headers.
```
Comment (ingo) [inline content/en/docs/integrate_with_central/platform-auth-examples.md:12]: HTTP in caps (likely, check style guide) and  extra space between http and request

File: content/en/docs/integrate_with_central/platform-auth-examples.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+## Create your service account via the CLI
+
+A service account is an Amplify concept that allows for a non-user, such as a CLI or headless process to gain access to the platform services and will be granted specific roles and privileges within on organization.
```
Comment (ingo) [inline content/en/docs/integrate_with_central/platform-auth-examples.md:16]: such as a CLI or headless process, to gain... (add comma)

Comment (ingo) [reply content/en/docs/integrate_with_central/platform-auth-examples.md:16]: *within an organization*

File: content/en/docs/integrate_with_central/platform-auth-examples.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+#### Using the CLI Directly
+
+##### Capture the Auth Result as a JSON Body
+
+```sh
+authResult=$(axway auth login --client-id test-sa-ccc_6d66dc36-f838-4006-8c44-5340d4698be5 --client-secret thisisasecret --json)
+```
+
+The command above will fulfill the authorization flow and cause the client ID and Secret to be base64 encoded and then passed to the auth server and then subsequently use the token to call platform services. (The alternative, manual steps, are outlined below for clarity)
```
Comment (ingo) [inline content/en/docs/integrate_with_central/platform-auth-examples.md:58]: ID is capitalized here but org id was lower-case. Likely a style guide check

Comment (ingo) [reply content/en/docs/integrate_with_central/platform-auth-examples.md:58]: Parenthetical expression should be inside the period.

Comment (ingo) [reply content/en/docs/integrate_with_central/platform-auth-examples.md:58]: The alternative--manual steps, are outlined below...

File: content/en/docs/integrate_with_central/platform-auth-examples.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+##### Extract the Bearer Token and TenantID
+
+```sh
+token=$(echo $authResult | jq -r '.auth.tokens.access_token')
+tenantId=$(echo $authResult | jq -r '.org.id')
+```
+
+#### Alternative approach call auth server directly
+
+This approach is more cumbersome, but demonstrates what is necessary if you decide to build against an application using a language like JavaScript, Java or Golang.
```
Comment (ingo) [inline content/en/docs/integrate_with_central/platform-auth-examples.md:69]: "against an application"?

**Review by ingo — COMMENTED** [review]
Minor grammar changes. Suggest also having Leor and @bladedancer take a look.

**Comment by lbadenhop** [comment]
@chrisccoy, please let me know when this is ready for my review (after Leor and @bladedancer). Thanks.

**Comment by bladedancer** [comment]
LGTM

**Comment by chrisccoy** [comment]
@lbrenman OK, I think I have it dialed in as we discussed on teams. Please give it a look and let me know if you'd like to see anything additional.

**Comment by lbrenman** [comment]
@chrisccoy - this looks great. Thanks for incorporating my feedback!

**Comment by chrisccoy** [comment]
> @chrisccoy - this looks great. Thanks for incorporating my feedback!

Easy when someone has already done all the leg work. Thanks to you!

**Review by lbadenhop — APPROVED** [review]

**Comment by lbadenhop** [comment]
@chrisccoy Please review my edits before I merge. Thanks!

**Comment by chrisccoy** [comment]
> @chrisccoy Please review my edits before I merge. Thanks!

Looks great, thanks

**Pull request merged**